docs/source/posts/2022/2022-09-20-week-13-mohamed.rst

```diff
@@ -0,0 +1,41 @@
+Week 13: Keyframes animation is now a bit easier in FURY
+========================================================
+
+.. post:: September  20 2022
```

Comment:
Idk how, but my eyes caught up the extra space after the `September`.
I am not sure whether it would cause any error or not but it would be great if you could remove that.